packages/mgt-element/src/IGraph.ts

```diff
@@ -6,6 +6,7 @@
  */
 
 import { Client, GraphRequest } from '@microsoft/microsoft-graph-client';
+import { GRAPH_URLS } from '@microsoft/microsoft-graph-client/lib/es/src/Constants';
```

Comment:
Why are we adding it but never using it?

Reply:
Oops...
```suggestion
```